exercises/01.advanced-tools/02.problem.structured/src/tools.ts

```diff
@@ -41,6 +41,9 @@ export async function initializeTools(agent: EpicMeMCP) {
 				}
 			}
 
+			// 🐨 you'll need to get the entry with the tags added
+			// 💰 const entryWithTags = await agent.db.getEntry(createdEntry.id)
+			
```

Comment:
We don't do this in [the solution](https://github.com/epicweb-dev/advanced-mcp-features/blob/fb068ae131a6c4a320ae65a1c1b0cf6c8296c058/exercises/01.advanced-tools/02.solution.structured/src/tools.ts#L44) 🤔

I want to merge your other fixes though so I'm going to remove this from this PR. If I'm wrong, let me know!

```suggestion
```

Reply:
Ah yeah, it's a bug in the solution code. I could submit a separate PR for that, to update both problem and solution files, I documented the bug in the issue.